src/main/java/core/basesyntax/cages/Aquarium.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.cages;
+
+import core.basesyntax.types.Animal;
+import core.basesyntax.types.Swimmable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Aquarium extends Zoo {
```

Comment:
```suggestion
public class Aquarium<T extends Swimmable extends Zoo {
```
private List<T> fishesInAquarium
краще генералізувати весь клас, тоді в пут ти будеш ставити Т а не енімал і кастити його до Swimmable 
аналогічно з CageForBirds

Reply:
мені не вдається тоді генералузвати метод put() в загальному класі Zoo, від якого наслідуються всі клітки.  В Zoo я б хотів мати універсальні методи, по додавані тварин і годуванню, відповідно щоб методи були універсальними я вних передаю загальний батьківський клас всіх тварин Animal. Мені потрібно якось переробити структуру? @RomanDubovskyi